scripts/github_pr/github_pr.rb

```diff
@@ -0,0 +1,405 @@
+#!/usr/bin/ruby
+
+require 'octokit'
+require 'optparse'
+require 'yaml'
+require 'json'
+
+class GithubAPI
+  def get_client
+    client = Octokit::Client.new(:netrc => true)
```

Comment:
... `(netrc: true)`

Reply:
I used old notation as I assumed that we still had SLE11 workers in the CI. But the last workers were already switched. So switching to new notation.